ow_plexil/CMakeLists.txt

```diff
@@ -1,23 +1,49 @@
 cmake_minimum_required(VERSION 3.0.2)
+
+# Top-level build file for the PLEXIL executive, conditioned to build
+# one customized for either OceanWATERS or JPL's OWLAT testbed.
+
 project(ow_plexil)
 
-## Find catkin macros and libraries
-## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
-## is used, also find other catkin packages
-find_package(catkin REQUIRED COMPONENTS
-  roscpp
-  rospy
-  roslib
-  actionlib
-  actionlib_msgs
-  geometry_msgs
-  ow_lander
-  ow_faults_detection
-  owl_msgs
-  rqt_gui_py
-  message_generation
-  std_msgs
-)
+OPTION(OWLAT "Build to support JPL's OWLAT simulator instead of OceanWATERS" OFF)
+
+# Find the packages that are used by both OceanWATERS and OWLAT.
+
+if(OWLAT)
+  find_package(catkin REQUIRED COMPONENTS
```

Comment:
I suggest keeping a top-level find_package line for all packages common to both compile modes. You can put smaller conditional find_package lines right after it.

Reply:
I had tried doing this originally, and just tried for a quite a while again, but I always end up with `ow_lander` action header files not found.  It stumps me.  We can look at this in Dev Day.

Reply:
I tried these, and some other things, no luck.  I'm not sure this is worth more effort...

Reply:
Just tried the same thing again and it didn't work. I must have gotten lucky before; sometimes things get built in a different order. I recommend making a ticket to rename `owl_msgs` to prevent collisions and remove `list(INSERT CMAKE_PREFIX_PATH 0 "/opt/ros/noetic/share")`, and make another ticket to fix this dependency issue. It will almost certainly appear again sometime later.

Reply:
Created OCEANWATER-1150 to rename/remove `owl_msgs`.